katas/content/nonlocal_games/chsh_quantum_bob_strategy/solution.md

```diff
@@ -0,0 +1,11 @@
+Measuring a qubit in the $\theta$ basis is the same as rotating the qubit by $\theta$, clockwise, and then making a standard measurement in the Z basis.
+
+To implement the described transformation in Q#, we need to rotate the qubit by $\frac{\pi}{8}$ clockwise if `bit == false` or counterclockwise if `bit == true` and then perform a measurement.
+We can do the rotation using the previous task (note the negation of the boolean parameter we need to do).
```

Comment:
Since we merged the two tasks, we can't refer to the previous task any longer
```suggestion
We can do the rotation using the Ry gate (note the negation of the Boolean parameter we need to do).
```

Reply:
Updated. Macro R_y is used instead of "Ry gate"